cmd/state/verify/check_indexes.go

```diff
@@ -21,32 +22,54 @@ func CheckIndex(ctx context.Context, chaindata string, changeSetBucket string, i
 	}
 	defer tx.Rollback()
 
+	fmt.Printf("Checking index %s against changeset %s\n", indexBucket, changeSetBucket)
 	startTime := time.Now()
 
 	i := 0
-	if err := changeset.ForEach(tx, changeSetBucket, nil, func(blockN uint64, k, v []byte) error {
+
+	var blockNumBytes []byte
+
+	if startBlock > 0 {
+		log.Info("Processing", "blockNum", startBlock)
+		blockNumBytes = dbutils.EncodeBlockNumber(startBlock)
+	}
+
+	if len(blockNumBytes) > 0 {
+		fmt.Printf("Using startkey[] %v\n", common.Bytes2Hex(blockNumBytes))
+	}
+
+	if err := changeset.ForEach(tx, changeSetBucket, blockNumBytes, func(blockN uint64, k, v []byte) error {
 		i++
 		if i%100_000 == 0 {
-			fmt.Printf("Processed %dK, %s\n", blockN/1000, time.Since(startTime))
+			log.Info("Processed", "changes", i, "block", blockN, "changeset", changeSetBucket, "index", indexBucket)
+		} else if i == 1 {
+			fmt.Printf("First change: block %v, key %v\n", blockN, common.Bytes2Hex(k))
 		}
+
 		select {
 		default:
 		case <-ctx.Done():
-			return ctx.Err()
+			err2 := ctx.Err()
+			log.Info("Done", "ctx.Err", err2)
+			return err2
```

Comment:
FYI: you cann add all this context to error message by: fmt.Errorf(“ctxDone: %w”)

Reply:
Can add any variables to error message same way

Reply:
> FYI: you cann add all this context to error message by: fmt.Errorf(“ctxDone: %w”, err2)

I think the original reason I printed it there is was because the caller didn't print out the returned error value and I was trying not to disturb the original behavior.

Thanks for suggestion on make lintci-deps and make lint, I will do that in the future before sending any additional PRs.